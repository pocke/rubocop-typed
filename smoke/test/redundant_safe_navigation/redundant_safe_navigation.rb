arr = [1, 2, 3]

elm = arr.find {|x| x == 1}

# unnecessary safe navigation operator
arr&.first

# It is necessary
elm&.chr

