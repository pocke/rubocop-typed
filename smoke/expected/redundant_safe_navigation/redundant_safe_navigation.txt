C:  6:  1: Typed/RedundantSafeNavigation: Do not use safe navigation operator with non nilable expression.
